snmp/metadata.csv

```diff
@@ -559,7 +559,7 @@ snmp.cyberpower.ePDUStatusInputVoltage,gauge,,,,"[CPS-MIB] The input voltage in
 snmp.cyberpower.ePDUStatusOutlet,gauge,,,,"[CPS-MIB] Allows for getting of status of each Outlet of an Outlet Monitored Rack PDU. (Make 'sum by {X}' queries to count elements with the tag X.)",0,snmp,,
 snmp.cyberpower.ePDUStatusPhase,gauge,,,,"[CPS-MIB] Allows for getting of status of each Rack PDU Phase (Make 'sum by {X}' queries to count elements with the tag X.)",0,snmp,,
 snmp.cyberpower.envirHumidity,gauge,,,,"",0,snmp,,
-snmp.cyberpower.envir_temperature,gauge,,,,"",0,snmp,,
+snmp.cyberpower.envirTemperature,gauge,,,,"[CPS-MIB] The environment temperature expressed in 1/10 Fahrenheit",0,snmp,,
```

Comment:
Any reason not to fix the missing description for envirHumidity as well? It should read "[CPS-MIB] The environment relative humidity expressed in percentage"